source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+- For the listening, thinking, replying: use a light background
+- For error: As you like
+  
+You can also save the images with transparent background and then change the background in the configuration file.
+
+#### Image dimensions and file format
+
+- **Screen dimensions**: 320 x 240 pixels
+- **File format**: png
+
+### To prepare your own images
+
+1. Create your own images according the specifications defined [above](#about-the-images).
+2. Paste all 6 images into a folder. For example: `frenck`.
+3. Make sure you have [access to your configuration files](/common-tasks/os/#configuring-access-to-files).
+   - [Install the Samba add-on](/common-tasks/os/#installing-and-using-the-samba-add-on).
+   - Samba allows you to copy paste multiple files at once.
+4. Paste your image folder into the configuration folder:
+   - Using Samba, open the `config` folder and open the `ESPHome` folder.
+   - Paste your image folder in there.
+   ![ESP32-S3-BOX config file](/images/assist/s32-s3-add-image-folder.png)
+
+### To add your images to the configuration
+
+1. Go to **Settings** > **Add-ons**, select the **ESPHome** add-on and **Open Web UI**.
+2. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+
+3. For this tutorial, we will use some images of Frenck.
+   - To add your images, add the following lines into the `substitution` block.
+    {% raw %}
+
+     ```yaml
+      loading_illustration_file: /config/esphome/frenck/loading_320_240.png
+      idle_illustration_file: /config/esphome/frenck/idle_320_240.png
+      listening_illustration_file: /config/esphome/frenck/listening_320_240.png
+      thinking_illustration_file: /config/esphome/frenck/thinking_320_240.png
+      replying_illustration_file: /config/esphome/frenck/replying_320_240.png
+      error_illustration_file: /config/esphome/frenck/error_320_240.png
+      ```
+
+    {% endraw %}
+
+4. If you used transparency in your images and you want to change the background color, add the following lines:
+   - The default is black and white.
+   - If you want to use different colors, replace the corresponding hexadecimal color code.
+     {% raw %}
+
+     ```yaml
+      loading_illustration_background_color: '000000'
+      idle_illustration_background_color: '000000'
+      listening_illustration_background_color: 'FFFFFF'
+      thinking_illustration_background_color: 'FFFFFF'
+      replying_illustration_background_color: 'FFFFFF'
+      error_illustration_background_color: '000000'
+      ```
+
+   {% endraw %}
+5. Save the changes and select **Install**.
+   ![ESP32-S3-BOX config file](/images/assist/s32-s3-add-image-config-02.png)
+6. Save the changes and restart Home Assistant.
+   - Depending on your environment, the installation process can take a while.
+   - Expect this to take about 25 minutes. The code needs to be compiled before it can be installed.
```

Comment:
Should we explain the word "compilation" or can we just talk about creating your customized voice assistant ?

Reply:
Also, subsequent installs are faster.